tests/RobotFramework/tests/cumulocity/custom_operation/custom_operation_timeout/custom_operation.robot

```diff
@@ -12,15 +12,15 @@ Test Tags           theme:c8y    theme:operation    theme:custom
 *** Test Cases ***
 Custom operation successful
     ThinEdgeIO.Transfer To Device    ${CURDIR}/c8y_Command_Success    /etc/tedge/operations/c8y/c8y_Command
-    ThinEdgeIO.Restart Service    tedge-mapper-c8y
+    Sleep    2s    reason=Give the mapper a chance to detect the change
```

Comment:
Wouldn't that (updated supported operations list) be better?
```suggestion
    Should Have MQTT Messages
    ...    c8y/s/us
    ...    message_pattern="114,.*c8y_Command,.*"
```

Reply:
Yeah good suggestion, for the first test this should be fine. I'll have to check if the same check will work for the second test given that we're only updating the contents (and not changing the file name), so I'm not sure if the known operations list is cached or not...but it should be easy enough to check.

Reply:
Forget what I just said.. @albinsuresh you're 100% right...for some reason I thought the test was reusing the same container (it is not), and testing that the contents of the custom operation handler was being tested (it isn't)...so you're proposal is spot on, and I'll implement that now.

Reply:
I used a slight variant of your suggestion. I opted for using the c8y interface `Should Contain Supported Operations` rather than relying knowing the cloud specific c8y topic.

```
Should Contain Supported Operations    c8y_Command
```